### ISSUE TITLE
Update code to not load gi for .wav files in Linux

### ISSUE THREAD

**Issue by pybokeh** [pvcraven/arcade#213]
## Bug Report
File "/home/pybokeh/envs/gamedev/lib/python3.6/site-packages/arcade/sound.py", line 133, in _playsound_unix
    import gi

ModuleNotFoundError: No module named gi

### Actual behavior:
Crashed and then I get ModuleNotFoundError: No module named gi

### Expected behavior:
Play a .wav sound without crashing

### Steps to reproduce/example code:
python -m arcade.examples.asteroid_smasher

**Comment by jgrantdev** [comment]
I get the same error that you do. Installing the below system packages and the python package got it working for me.

```
sudo apt-get install libcairo2-dev libgirepository1.0-dev
pip install pygobject
```
When you `pip install pygobject` it may fail, there could be more system packages required that I already had on my system.

Let me know if this works for you so I can update the documentation. Also it appears to freeze the game when the sound is being played, I am unsure if this is the behavior on other platforms as well.

**Comment by pvcraven** [comment]
Gstreamer doesn't work with venv. But it isn't needed for .wav files. I'll update the code so it try loading it for .wav.

**Comment by pybokeh** [comment]
@jgrantdev I installed those things, the asteroid smasher example works and plays sound, but when it plays the sound it lags or freezes for a second or so.

**Comment by pvcraven** [comment]
Ok, I'll look into that too.

**Comment by pvcraven** [comment]
Closed with release of 1.3.4 on 28-May-2018.
